.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,26 @@
+name: coverage
+on:
+  pull_request:
+    branches:
+    - main
+  push:
+    branches:
+    - main
+jobs:
+  run_unit_tests:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - run: docker-compose -f ./docker-compose.yml up --detach redis
+    - uses: actions/setup-python@v3
+      with:
+        python-version: "3.10.x"
+    - run: source ./activate.sh
+    - run: pip install --upgrade pip
+    - run: pip install -e .
+    - run: pip freeze
+    - run: ./test.sh
+    - uses: codecov/codecov-action@v3.1.0
+      with:
+        token: e1038ca3-70ae-433d-be18-5764f119c744
```

Comment:
Need to change this token since this belongs to the fork `cglotr/aiotaskq`.

Reply:
file:///home/cglotr/Pictures/Screenshots/Screenshot%20from%202022-06-10%2004-25-11.png